### PR TITLE
SEO-171754 WPF Redirect

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @Aishwarya-Ganesan ,

Have made the changes in redirect links. Please review.

Thanks.
Regards,
Edith.

### CODE REVIEW

File: wpf/DataGrid/Column-Types.md [hunk 1]
```diff
@@ -2758,7 +2758,7 @@ public class GridCellTextBoxRendererExt:GridCellTextBoxRenderer
 
 ### Create the renderer of existing column
 
-You can change the renderer of existing column by removing the predefined cell type value from [CellRenderers](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_CellRenderers) collection and add the newly derived renderer from [GridVirtualizingCellRenderer](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridVirtualizingCellRenderer%602.html).
+You can change the renderer of existing column by removing the predefined cell type value from [CellRenderers](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_CellRenderers) collection and add the newly derived renderer from [GridVirtualizingCellRenderer](https://help.syncfusion.com/cr/wpf/Syncfusion.html).
```
Comment (gowthamanthavasiyappan) [inline wpf/DataGrid/Column-Types.md:2761]: Can you check and confirm this redirection link with the product team?

File: wpf/DataGrid/Column-Types.md [hunk 2]
```diff
@@ -3015,7 +3015,7 @@ public class DisplayConverter : IValueConverter
 
 ### Creating new column and renderer
 
-You can create a new column by deriving [GridColumn](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html), rendered in UI using customized [CellType](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html#Syncfusion_UI_Xaml_Grid_GridColumn_CellType) using [GridVirtualizingCellRenderer](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridVirtualizingCellRenderer%602.html).
+You can create a new column by deriving [GridColumn](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html), rendered in UI using customized [CellType](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GridColumn.html#Syncfusion_UI_Xaml_Grid_GridColumn_CellType) using [GridVirtualizingCellRenderer](https://help.syncfusion.com/cr/wpf/Syncfusion.html).
```
Comment (gowthamanthavasiyappan) [inline wpf/DataGrid/Column-Types.md:3018]: Recheck this link also.

File: wpf/DataGrid/Grouping.md [hunk 3]
```diff
@@ -601,10 +601,10 @@ The [GroupChangingEventArgs](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xa
 
 [Group](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.GroupChangingEventArgs.html#Syncfusion_UI_Xaml_Grid_GroupChangingEventArgs_Group) - Gets the group that’s being collapsed.
 
-[Cancel](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-5.0) – Decides whether to cancel the group collapsing.
+[Cancel](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-5.0) – Decides whether to cancel the group collapsing.
 
  
-You can cancel the group is being collapsed by using [GroupChangingEventArgs.Cancel](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-5.0) of `GroupCollapsing` event.
+You can cancel the group is being collapsed by using [GroupChangingEventArgs.Cancel](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.SfDataGrid.html#Syncfusion_UI_Xaml_Grid_SfDataGrid_ShowGroupDropArea) of `GroupCollapsing` event.
```
Comment (gowthamanthavasiyappan) [inline wpf/DataGrid/Grouping.md:607]: Redirection link is wrong, update proper link here.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: wpf/DataGrid/Column-Types.md (hunk 2 above)
Comment (Edithabich) [reply wpf/DataGrid/Column-Types.md]: I have changed the redirect link here

File: wpf/DataGrid/Grouping.md (hunk 3 above)
Comment (Edithabich) [reply wpf/DataGrid/Grouping.md]: Have changed the proper link here

**Pull request merged**